packages/contentstack-auth/README.md

```diff
@@ -17,8 +17,8 @@ It is Contentstack’s CLI plugin to perform authentication-related activities.
 $ npm install -g @contentstack/cli-auth
 $ csdx COMMAND
 running command...
-$ csdx (-v|--version|version)
-@contentstack/cli-auth/1.0.0 darwin-x64 node-v16.15.1
+$ csdx (--version)
+@contentstack/cli-auth/1.0.0 linux-x64 node-v16.14.2
```

Comment:
Why are we not using the latest LTS here?
Why are we bringing down the node version from 16.15.1 to 16.14.2, I can see latest LTS is at 16.17.0?

Reply:
done